### PR TITLE
Rename peg-in/peg-out to deposit/withdrawl

### LINKED ISSUE #654: Rename peg-out and peg-in contracts to their new names.
Rename peg-out and peg-in contracts to their new names.

### PR DESCRIPTION
**Pull request by friedger**
This PR
* renames sbtc-peg-in-processor and sbtc-peg-out-processor and corresponding names in clarity code
* renames sbtc-peg-transfer to sbtc-hand-off
* renames peg-wallet to sbtc-wallet
* fixes #654

### CODE REVIEW

**Comment by FriendsFerdinand** [comment]
Typos: withdrawl is spelled withdrawal.

**Review by FriendsFerdinand — CHANGES_REQUESTED** [review]
Typos: withdrawl should be spelled withdrawal.

File: sbtc-mini/Clarinet.toml [hunk 1]
```diff
@@ -35,18 +35,18 @@ path = 'tests/sbtc-controller_test.clar'
...
-[contracts.sbtc-peg-in-processor_test]
-path = 'tests/sbtc-peg-in-processor_test.clar'
+[contracts.sbtc-deposit-processor_test]
+path = 'tests/sbtc-deposit-processor_test.clar'
 clarity_version = 2
 epoch = 2.4
 
-[contracts.sbtc-peg-out-processor]
-path = 'contracts/sbtc-peg-out-processor.clar'
+[contracts.sbtc-withdrawl-processor]
```
Comment (setzeus) [inline sbtc-mini/Clarinet.toml:48]: As mentioned by @FriendsFerdinand , only issue I see is that "withdrawl" needs to be universally updated to "withdrawal."

Outside of that, everything else LGTM.

**Review by setzeus — CHANGES_REQUESTED** [review]

File: sbtc-mini/Clarinet.toml (hunk 1 above)
Comment (friedger) [reply sbtc-mini/Clarinet.toml]: Typo fixed

**Review by FriendsFerdinand — APPROVED** [review]

**Review by setzeus — APPROVED** [review]
Withdrawal spelling issue fixed, all unit tests intact - LGTM!

**Comment by MarvinJanssen** [comment]
I wish you could have waited with this. Peg out is still in flight and it'll be headaches merging now. 

Anyway, appreciate you proactively taking on this chore.

**Pull request merged**